rocketpy/monte_carlo/mc_solid_motor.py

```diff
@@ -138,18 +208,21 @@ def __init__(
             center_of_dry_mass_position=center_of_dry_mass_position,
             nozzle_position=nozzle_position,
             throat_radius=throat_radius,
+            interpolate=None,
+            coordinate_system_orientation=None,
```

Comment:
I think nobody will ever change the `coordinate_system_orientation` in a Monte Carlo loop, but I'm not sure what were your intentions here so just double check it please.

Reply:
Yep, these lines are guaranteeing that the interpolation and coordinat_system_orientation will be gotten from the original object. Note that these arguments are not on the class' constructor